pallets/order-book/src/lib.rs

```diff
@@ -457,31 +457,11 @@ pub mod pallet {
             order_book_id: OrderBookId<AssetIdOf<T>, T::DEXId>,
         ) -> DispatchResult {
             let who = ensure_signed(origin)?;
-            ensure!(
-                order_book_id.base != order_book_id.quote,
-                Error::<T>::ForbiddenToCreateOrderBookWithSameAssets
-            );
             ensure!(
                 order_book_id.dex_id == common::DEXId::Polkaswap.into(),
                 Error::<T>::NotAllowedDEXId
             );
```

Comment:
why didn't you move it into `verify_create_orderbook_params`?

Reply:
I'm not certain that it shouldn't be in `verify_create_orderbook_params`, so I can move it here if necessary

Reply:
It is a part of this check